client/app/locales/locale-de.json

```diff
@@ -23,7 +23,8 @@
     "REDIRECT": "Umleitung zu deiner Gruppe..."
   },
   "TOPBAR": {
-    "LOGOUT": "Ausloggen"
+    "LOGOUT": "Ausloggen",
+    "GREETING": "Moin, {{username}}!"
```

Comment:
We always keep the english version though, right? So it just doesn't make sense to translate into other languages?